.github/workflows/aidlc-workflows-docs-update.yml

```
@@ -92,11 +114,12 @@ jobs:
 
       # Cheap gate: one ls-remote, no clone. Runs daily and does nothing on
       # almost every run.
-      - name: Compare the pinned SHA against the upstream v2 tip
+      - name: Compare the pinned SHA against the upstream tip
         id: compare
         shell: bash
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          UPSTREAM_BRANCH: ${{ steps.compare.outputs.upstream_branch }}
```

actionlint (1.7.12)

[error] property "compare" is not defined in object type {}

(expression)